contrib/camel/samples/README.md

```diff
@@ -0,0 +1,65 @@
+# Camel Source
+
+These samples show how to configure a Camel source. It is a event source that
+can leverage one of the [250+ Apache Camel components](https://github.com/apache/camel/tree/master/components)
+for generating events.
+
+## Deployment Steps
+
+These steps assume that you have checked out the repo and have a shell in this
+`samples` directory.
+
+### Prerequisites
+
+1. Install the
```

Comment:
Should this direct users to install camel-k from https://github.com/apache/camel-k in addition to this Camel Source? Right now, it looks like 3-5 files have been vendored in from that directory with some modifications.

Reply:
We wanted to make it easy for people to write camel sources, so we bundled everything needed to run them in the yaml files contained in this repo, no additional steps needed. I think this allows a better user experience.. The files vendored here are the minimal set of objects needed for the controller manager to hand objects over to the camel k operator.